sdk/identity/identity/README.md

```diff
@@ -1,3 +1,99 @@
-# Azure SDK Identity Library
+# Azure SDK Identity Library for C
 
 Azure Identity library (`az_identity`) provides primitives, abstractions, and helpers for Azure authorization.
+
+## Getting started
+
+### Install the package
+
+TODO link to the vcpkg
+
+**Prerequisites**: You must have an [Azure subscription][azure_sub] and [C++ compiler][cpp_compiler] to use this package.
```

Comment:
This says C++ compiler; but C required.

Reply:
Switched to c and added link to c compiler